hs_core/hydroshare/hs_bagit.py

```diff
@@ -7,6 +7,10 @@
 from uuid import uuid4
 from mezzanine.conf import settings
 
+from django.core.exceptions import ValidationError
+
+from django_irods.icommands import SessionException
```

Comment:
@ismagilani Line 12 import above is already in develop branch as shown here: https://github.com/heliumdatacommons/commonsshare/blob/develop/hs_core/hydroshare/hs_bagit.py#L13, so I believe this is a duplicate import that can be removed from your branch.